src/main/java/org/cirdles/topsoil/app/TopsoilMainWindow.java

```diff
@@ -235,7 +278,8 @@ public void changed(ObservableValue<? extends Scene> observableScene, Scene oldS
     }
 
     public TSVTable createTable() {
-        return (TSVTable) createTab().getContent();
+        TabContents content = (TabContents) createTab().getContent();
+        return content.getTable().get();
```

Comment:
This has to be rewritten for `TabContents`. `createInstructionsTab` and this should share the logic in `createTab`. Here, `createTab` is creating the table.